client/src/resources/cert-and-project-map.ts

```diff
@@ -769,23 +773,52 @@ function getJavaScriptAlgoPath(project: string) {
     : `${jsAlgoBase}/${project}`;
 }
 
-const titles = certMap.map(({ title }) => title);
-type Title = (typeof titles)[number];
-const legacyProjectMap: Partial<Record<Title, unknown>> = {};
-const projectMap: Partial<Record<Title, unknown>> = {};
+const certMapWithoutFullStack = Array.of(
+  ...upcomingCertMap,
+  // @ts-expect-error TODO: This did not used to be a type error
```

Comment:
I cannot explain this one. This error was not here, before I pulled upstream.

So, either TS was going crazy on my machine. Or, we have changed TS config such that this now produces an error.

Reply:
Turns out the solution was to just use `[...upcomingCertMap, ...]` instead of `Array.of`